packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc

```diff
@@ -7,63 +7,65 @@
 #include "tizen_app_manager.h"
 
 static bool AppInfoCB(app_info_h app_info, void *user_data) {
```

Comment:
> Consider inlining a C-style callback using a lambda expression if it makes the code cleaner.

we prefer lambda instead a C-style callback.

Reply:
the callback function registration method using lambda has been changed to use.